building plugin doc fails on Mac OS X

When running the command `cmake .. -DPLUGINS='dump;resolver;sync;error;doc'`, `make` fails with the following message:

```
[ 19%] Linking C shared module ../../../lib/libelektra-doc.so
Undefined symbols for architecture x86_64:
  "_kdbSet", referenced from:
      _elektraUsercodeUselessSymbol in doc.c.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [lib/libelektra-doc.so] Error 1
make[1]: *** [src/plugins/doc/CMakeFiles/elektra-doc.dir/all] Error 2
make: *** [all] Error 2
```

--- a/src/plugins/doc/CMakeLists.txt
+++ b/src/plugins/doc/CMakeLists.txt
@@ -3,4 +3,7 @@ include(LibAddMacros)
 add_plugin(doc
 	SOURCES
 		doc.h doc.c
-	)
+	LINK_ELEKTRA
+		elektra-kdb
+)
+

